lib/roda/plugins/exception_page.rb

```diff
@@ -84,11 +96,20 @@ module RodaPlugins
     # Used under the modified BSD license:
     # http://www.xfree86.org/3.3.6/COPYRIGHT2.html#5
     module ExceptionPage
-      def self.load_dependencies(app)
+      def self.load_dependencies(app, opts=OPTS)
         app.plugin :h
       end
 
-      # Stylesheet used by the HTML exception page
+      # Use the CSS given as the :css option as the exception page's CSS and the
+      # JavaScript given as the :js option as the exception page's JavaScript,
+      # falling back to the defaults when not given.
+      def self.configure(app, opts=OPTS)
+        css, js = opts.fetch(:assets, NOOP).call(+self.css, +self.js)
+        app.exception_page_css = -css
+        app.exception_page_js = -js
```

Comment:
The `+`/`-` dance here is to make it so we don't end up shuffling more memory by `dup`ing the frozen strings defined by `ExceptionPage.css` and `ExceptionPage.js`. I know they're still `dup`ed but they're then deduped when unmodified.

I could make this a longer method to eliminate the allocation entirely, if you like the approach and would prefer that.

Reply:
You cannot use `String#{@+,@-}` . Roda supports Ruby 1.9+.